src/main/java/tempest/ui/pages/DataProtectionPage.java

```diff
@@ -0,0 +1,42 @@
+package tempest.ui.pages;
+
+import tempest.State;
+import tempest.ui.GUIManager;
+import tempest.ui.components.BackButton;
+
+import javax.swing.*;
+
+public class DataProtectionPage extends Page {
+    private BackButton backButton;
+    static String text;
+    public DataProtectionPage(State state, GUIManager manager) {
+        super(manager);
+        setupUI();
+    }
+
+    @Override
+    public String getName() {
+        return PageNames.DATA_PROTECTION;
+    }
+    private void setupUI(){
+        JPanel backPanel = new JPanel();
+        JPanel textPanel = new JPanel();
+        text = "<ul><li>All data collected is stored in a CSV file called store.csv\n" +
+                "<li> Any data inputted by the user from within the program will be stored within this file\n" +
```

Comment:
`Any data inputted by the user from within the program will be stored within this file`

This won't be the case when Josh's branch is pushed (as weekly goals for each module aren't saved to the csv) so could you change this line to talk about this